gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/TestWSGremlinInitializer.java

```diff
@@ -136,5 +144,28 @@ private String returnSingleVertexResponse(final UUID requestID) throws Serializa
 
             return SERIALIZER.serializeResponseAsString(ResponseMessage.build(requestID).result(t).create());
         }
+
+        private String returnSimpleStringResponse(final UUID requestID, String message) throws SerializationException {
+            return SERIALIZER.serializeResponseAsString(ResponseMessage.build(requestID).result(message).create());
+        }
+
+        /**
+         * Captures and stores user-agent if included in header
+         */
+        @Override
+        public void userEventTriggered(ChannelHandlerContext ctx, java.lang.Object evt){
```

Comment:
better to mark params `final`
```suggestion
        public void userEventTriggered(final ChannelHandlerContext ctx, final Object evt){
```